main/include/eudaq/DataCollector.hh

```diff
@@ -56,14 +58,18 @@ namespace eudaq {
                                    //       pthread_attr_t m_threadattr;
     std::unique_ptr<std::thread> m_thread;
     std::vector<Info> m_buffer;
-    size_t m_numwaiting; ///< The number of producers with events waiting in the
-                         ///buffer
+
     size_t m_itlu;       ///< Index of TLU in m_buffer vector, or -1 if no TLU
+    size_t m_slow;     /// The number of slow producers registered
     unsigned m_runnumber, m_eventnumber;
     std::shared_ptr<FileWriter> m_writer;
+
+    //std::vector<size_t> ireceived;    // storage of producers which sent event
```

Comment:
dead code --> if it is not needed remove, it is still in the git history available